<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResolveMultipleGroupsPage</class>
 <widget class="QWizardPage" name="ResolveMultipleGroupsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resolve Multiple Groups Page</string>
  </property>
  <property name="title">
   <string>Resolve Hardware conflict</string>
  </property>
  <property name="subTitle">
   <string>You have selected multiple groups of hosts with conflicting hardware properties. This wizard can resolve this in one of the following ways:</string>
  </property>
  <attribute name="pageId">
   <string notr="true">1</string>
  </attribute>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Hardware Property Set to be assigned to a Host Template</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="commonRadioButton">
          <property name="text">
           <string>only common Hardware Properties</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="frequentRadioButton">
          <property name="text">
           <string>most frequent Hardware Properties</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="customRadioButton">
          <property name="text">
           <string>custom Hardware Property Set</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="HWPropsTableWidget" name="propsTableWidget"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Handling of remaining Hardware Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="propPerHostRadioButton">
        <property name="text">
         <string>add Properties individually per host</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="setPerHostRadioButton">
        <property name="text">
         <string>add Property Set per host</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="laterRadioButton">
        <property name="text">
         <string>configure manually later</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HWPropsTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">HWPropsTableWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
